src/reconcile/rec_write.c

```diff
@@ -881,8 +873,8 @@ __rec_split_chunk_init(WT_SESSION_IMPL *session, WT_RECONCILE *r, WT_REC_CHUNK *
  *     Initialization for the reconciliation split functions.
  */
 int
-__wt_rec_split_init(
-  WT_SESSION_IMPL *session, WT_RECONCILE *r, WT_PAGE *page, uint64_t recno, uint64_t max)
+__wt_rec_split_init(WT_SESSION_IMPL *session, WT_RECONCILE *r, WT_PAGE *page, uint64_t recno,
+  uint64_t primary_size, uint64_t auxiliary_size)
```

Comment:
Everything else passes it btree->maxleafpage_precomp, which is uint64, so under the current circumstances it would need casts everywhere, so it's probably not a great idea. maxleafpage_precomp in turn comes from __rec_compression_adjust; it's not obvious why that's using uint64, but since it's doing unlocked writes changing it is probably best not done casually.

I can make auxiliary_size uint32 easily enough since everybody else passes 0. (fixed)